### PR TITLE
added few channels and logos

### LINKED ISSUE #4140: Tunes 6
**Channel Name:** Tunes 6
**Country:** IN
**Language:** Tamil
**Website:** Tunes6.com
**Stream URL (optional):** https://www.mxplayer.in/live-tv/tunes-6/selfie-request-30000000550204845-140
**Notes (optional):** xxx

### LINKED ISSUE #4202: Mazhavil Manorama SD
**Channel Name:** Mazhavil Manorama SD
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** http://cdn.asianetmobiletvplus.com/channels/mazhavil_wxjylngynrbeykzthdrhawtunzcuowsr/l_6568_47694648_7942.ts?nimblesessionid=24553390
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by Niyasmkkv**
Closes #4202 
Closes #4140

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -472,6 +475,10 @@ http://185.180.221.194:8278/streams/d/starvijay/playlist.m3u8
 https://tna5.bozztv.com/streamlive/index.m3u8
 #EXTINF:-1 tvg-id="SteelbirdMusic.in" tvg-name="Steelbird Music" tvg-country="IN" tvg-language="" tvg-logo="https://i.imgur.com/5r4gCMh.jpg" group-title="Music",Steelbird Music (720p) [Not 24/7]
 http://cdn25.live247stream.com/steelbirdmusic/tv/playlist.m3u8
+#EXTINF:-1 tvg-id="SudioOnePlus.in" tvg-name="Studio One Plus" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/DBbajAs.jpg" group-title="Movies",Studio One Plus
```
Comment (LaneSh4d0w) [inline channels/in.m3u:478]:
```suggestion
#EXTINF:-1 tvg-id="StudioOnePlus.in" tvg-name="Studio One Plus" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/DBbajAs.jpg" group-title="Movies",Studio One Plus
```

File: channels/in.m3u [hunk 2]
```diff
@@ -472,6 +475,10 @@ http://185.180.221.194:8278/streams/d/starvijay/playlist.m3u8
...
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCEoaVUxpFlmCxwRBy9yhb1Q/live
+#EXTINF:-1 tvg-id="SudioOneYuva.in" tvg-name="Studio One Yuva" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/nACKF3x.jpg" group-title="Movies",Studio One Yuva
```
Comment (LaneSh4d0w) [inline channels/in.m3u:480]:
```suggestion
#EXTINF:-1 tvg-id="StudioOneYuva.in" tvg-name="Studio One Yuva" tvg-country="IN" tvg-language="Telugu" tvg-logo="https://i.imgur.com/nACKF3x.jpg" group-title="Movies",Studio One Yuva
```

File: channels/in.m3u [hunk 3]
```diff
@@ -512,6 +519,8 @@ https://travelxp-travelxp-2-de.samsung.wurl.com/manifest/playlist.m3u8
 https://vidcdn.vidgyor.com/tv9telugu-origin/live/tv9telugu-origin/live1/playlist.m3u8
 #EXTINF:-1 tvg-id="TwentyFourNews.in" tvg-name="Twenty Four News" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/eS2fVDh.png" group-title="News",Twenty Four News (576p)
 http://103.199.160.85/Content/24news/Live/Channel(24news)/index.m3u8
+#EXTINF:-1 tvg-id="Tunes6.in" tvg-name="Tunes 6" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/ClpXKNR.png" group-title="Music",Tunes 6
```
Comment (LaneSh4d0w) [inline channels/in.m3u:522]:
```suggestion
#EXTINF:-1 tvg-id="Tunes6.in" tvg-name="Tunes 6" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/ClpXKNR.png" group-title="Music",Tunes 6 [Geo-blocked]
```

**Comment by Niyasmkkv** [comment]
@LaneSh4d0w Why KCV movies and Keralavision news are showing ofline? the link is working fine for me.

**Comment by LaneSh4d0w** [comment]
Either they're Not 24/7, either links were down on the time he looked at them, or there's a slight issue.

**Comment by Niyasmkkv** [comment]
> Either they're Not 24/7, either links were down on the time he looked at them, or there's a slight issue.

Will it be back if the link is working by the next check up time?

**Comment by LaneSh4d0w** [comment]
> 
> 
> > Either they're Not 24/7, either links were down on the time he looked at them, or there's a slight issue.
> 
> Will it be back if the link is working by the next check up time?

I cannot guarantee that.

**Comment by Niyasmkkv** [comment]
> > > Either they're Not 24/7, either links were down on the time he looked at them, or there's a slight issue.
> > 
> > 
> > Will it be back if the link is working by the next check up time?
> 
> I cannot guarantee that.

It shows offline for youtube links too. Channels like Sun News, Shekinah are seems offline in the list, but the link is working.

**Comment by LaneSh4d0w** [comment]
Maybe there's certain stuff the bot doesn't like

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**